custom_components/tuya_local/devices/greenmigo_alpha_q25_pro_wifi.yaml

```diff
@@ -0,0 +1,207 @@
+name: Greenmigo Alpha Q25 Pro WiFi Dehumidifier
```

Comment:
If possible to get the product id, it would be better to move this to the name under `products:` and use a generic name (ie just Dehumidifier).  But also, Home Assistant naming convention calls for non-name related parts to be lower case if not the first word (I think it only affects "Dehumidifier" here, as the rest is part of the branding).

Reply:
alright